86.md

```diff
@@ -0,0 +1,51 @@
+NIP-86
+=======
+
+Shared Keys
+-----------
+
+`draft` `optional` `author:staab` `author:earonesty` `author:vitorpamplona` `author:water783`
+
+A shared key allows key holders to encrypt and decrypt messages to all other key holders. A shared key is issued by a single pubkey (the `admin`) who has sole authority to issue invitations and key rotations.
+
+This NIP relies on [NIP 59](./59.md) for event wrapping.
+
+## Key Sharing
+
+Keys are shared via a `kind 24` rumor sealed by the `admin` key, wrapped using the shared key, and addressed to each recipient individually. The rumor MUST have a single `shared_key` tag containing the shared private key.
+
+```json
+{
+  "kind": 24,
+  "content": "Just a regular key rotation",
+  "tags": [["shared_key", "<new shared key>"]],
+}
```

Comment:
Shouldn't `kind: 24` tag which community is the shared secret for?

Reply:
Yep, just added that.